Update README.md (#244)
Zenika/feedzback: README.md

    Make sure you installed npm packages
 ### Environment Variables
    Create `.env` file and put those variables below in it:
    `MAILGUN_API_KEY`= your mailgun Api

>     `MAILGUN_DOMAIN`= your mailgun domain

Reviewer: Faudrait faire les modifs dans le code et dans le app.yaml du coup aussi

    `API_KEY`= your mailgun Api
    `DOMAIN`= your mailgun domain
    `URL_CLIENT_FORM` your url client form to set into email template

>     `DatastoreApi` yout endpoint datastore in order to simulate it in local

Reviewer 2: ```suggestion
    `DatastoreApi` your local datastore endpoint in order to simulate it in local
```

  ng test
 ### Run e2e tests
  npm run cypress
 ### Setup datastore emulator

>   setp 1: download it in your local

Reviewer 2: ```suggestion
  step 1: download the Firebase simulator
```
 Ça serait cool de rajouter l’URL de la documentation Firebase ici

>   gcloud components install cloud-datastore-emulator

Reviewer 2: ```suggestion
in your shell run `gcloud components install cloud-datastore-emulator`
```

>   step 2: run it
>   gcloud beta emulators datastore start

Reviewer 2: ```suggestion
  step 2: start the Firebase simulator
  in your shell run `gcloud beta emulators datastore start`
```

Author: on utilise datastore emulator
